a3fe/run/simulation.py

```diff
@@ -338,6 +302,8 @@ def run(self, runtime: float = 2.5) -> None:
         -------
         None
         """
+        #update somd.cfg with the necessary configurations
+        self._update_simfile()
 
         # Run SOMD - note that command excludes sbatch as this is added by the virtual queue
         cmd = f"somd-freenrg -C somd.cfg -l {self.lam} -p CUDA"
```

Comment:
The command should come from the engine config - please add some kind of method/ attribute to generate this.